src/neptune/types/series/float_series.py

```diff
@@ -96,3 +100,15 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def clear_of_unsupported_values(self):
+        cleared_values = []
+        for value in self._values:
+            if math.isinf(value) or math.isnan(value):
+                logger.warning(
+                    "WARNING: The value you're trying to log is a non-standard float value "
+                    "that is not currently supported. "
```

Comment:
Why don't we also include the value itself in the warning message? Might be a little easier to debug for a user ;)

Reply:
Added! 😇